android/samples/sample-gltf-viewer/src/main/java/com/google/android/filament/gltf/MainActivity.kt

```diff
@@ -65,6 +66,12 @@ class MainActivity : Activity() {
         val options = modelViewer.view.dynamicResolutionOptions
         options.enabled = true;
         modelViewer.view.dynamicResolutionOptions = options;
+
+        modelViewer.view.ambientOcclusion = View.AmbientOcclusion.SSAO
+
+        val bloom = modelViewer.view.bloomOptions
+        bloom.enabled = true;
+        modelViewer.view.bloomOptions = bloom
```

Comment:
Ok, I think I understand why couldn't do `modelViewer.view.bloomOptions.enabled = true`.  Maybe we should use the traditional Java-stye `setBloomOptions` function call here to show readers what's going on.

Reply:
I went through this with @romainguy before. As long as we do the same thing everywhere, I'm good. I so far we've done it this way.